lib/rules/no-unused-properties.js

```diff
@@ -453,8 +455,186 @@ module.exports = {
         }
       }),
       utils.defineVueVisitor(context, {
-        onVueObjectEnter(node) {
-          const container = getVueComponentPropertiesContainer(node)
+        /*
+          methods: mapMutations({
+            add: 'increment'
+          })
+
+          methods: {
+            ...mapMutations({
+              add: 'increment'
+            })
+          }
+        */
+        'CallExpression[callee.name=mapMutations][arguments] ObjectExpression Property[key.name]'(
```

Comment:
Can we somehow combine and simplify these cases? At lease object and array expression could be handled in the same visitor, maybe even all Vuex function calls.

Reply:
It is now better?